mmdeploy/apis/visualize.py

```diff
@@ -67,14 +67,23 @@ def visualize_model(model_cfg: Union[str, mmengine.Config],
         else:
             model = task_processor.build_backend_model(model)
 
-    model_inputs, _ = task_processor.create_input(img, input_shape)
+    model_inputs, _ = task_processor.create_input(
```

Comment:
the preprocessor should have been included in backend_model, we do not need to add it here.

Reply:
done.